mu4/scenes/inspector/models/score/scoredisplaysettingsmodel.cpp

```diff
@@ -0,0 +1,111 @@
+#include "scoredisplaysettingsmodel.h"
+
+ScoreSettingsModel::ScoreSettingsModel(QObject* parent, IElementRepositoryService* repository)
+    : AbstractInspectorModel(parent, repository)
+{
+    setSectionType(SECTION_SCORE_DISPLAY);
+    setTitle(tr("Show"));
```

Comment:
Please use `trc`

Reply:
I'm going to make another PR with the stuff like that. Because I need to add some missing copyrights, make some code-style fixes and so on.